README.md

```diff
@@ -79,6 +79,13 @@ Questions can be asked by raising an `Issue`.
 
 ```
   git push origin <name of your branch>
+
```

Comment:
```suggestion
```

Reply:
I am sorry , But i am not able to get you . Can you explain me a bit more please ?

Reply:
I think @schmelto wants to say that remove the unnecessary line 82 (There was no need to put enter on 82 & 88 line)

Reply:
Ok i got it . Thanks for the help. I will correct it